notebooks/3_KernelMethods.ipynb

```diff
@@ -70,7 +70,7 @@
     "except that we take advantage of the [**kernel trick**](https://en.wikipedia.org/wiki/Kernel_method#Mathematics:_the_kernel_trick) in order to \n",
     "introduce a non-linear transformation of the feature space.\n",
     "\n",
-    "The kernel trick consists in introducing a [**positive definite**](https://en.wikipedia.org/wiki/Positive-definite_kernel) function of pairs of samples, $ k(\\mathbf{x}, \\mathbf{x}^{\\prime})$, computed based on their features, that defines implicitly a higher (possibly infinite) dimensional feature space, in which one can apply linear analysis methods. \n",
+    "The kernel trick consists in introducing a [**positive definite**](https://en.wikipedia.org/wiki/Positive-definite_kernel) function of pairs of samples, $ k(\\mathbf{x}, \\mathbf{x}^{\\prime})$, computed based on their original features, that defines implicitly a higher (possibly infinite) dimensional feature space, in which one can apply linear analysis methods. \n",
```

Comment:
Why the addition of `original`?

Reply:
Because we have two kinds of features: the `original` feature and the `RKHS` feature. Adding `original` will help readers understand which feature we are referring to.
![image](https://user-images.githubusercontent.com/56018055/97762868-23d95280-1b0a-11eb-8e26-a677e4db61c8.png)
![image](https://user-images.githubusercontent.com/56018055/97762901-3d7a9a00-1b0a-11eb-9408-819ca61de153.png)

Reply:
I think I would actually omit "original" here. I think it is good to make the distinction between the original and the RKHS features, but since at this point in the text we haven't mentioned the RKHS features yet, I think the addition of "original" could cause confusion. We could also just remove the whole clause "computed based on their [original] features" and avoid the issue entirely